### PR TITLE
Self_Review_PR

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
In this PR, I did the following :

- Added HTML layout to all the three pages
- Added style to all three pages
- updated the READMe file

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @Jawad-Nazari , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that hasn't had any review kindly review your project by using inline comments so the author of the project can easily understand the context of your comment. Watch this [video with an example](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d). Also, Read this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for more information

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+.project-grid {
+  margin-top: 5%;
+  display: grid;
+  grid-gap: 4rem;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+}
+
+
```
Comment (Jawad-Nazari) [inline style.css:91]: There are some Linter errors on your code, so kindly use Linter before pushing your code to Github to fix any linter errors your code.
👍

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# Addmission-Trail-Jawad-and-Abiy
\ No newline at end of file
+# Addmission-Trail-Jawad-and-Abiy
+
+This is a project from Addmission trail that has done by Jawad and Abiy during the addmission process.
```
Comment (Jawad-Nazari) [inline README.md:3]: Your README file is great but It's better to use Microverse README file template and provide more information about the project and its features.
👍

**Review by Jawad-Nazari — COMMENTED** [review]
# Changes Required ♻️ 
Please, check my comments below and bring all required changes. and good luck.

**Comment by Jawad-Nazari** [comment]
Hi @aamirkhan2478 ,
Thanks for your time and for reviewing my review. I have added some inline comment and submit my code review request.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Jawad-Nazari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**